Dockerfile

```diff
@@ -44,8 +44,9 @@ ENTRYPOINT [ "/bin/eventsource" ]
 ####################################################################################################
 # sensor
 ####################################################################################################
-FROM centos:8 as sensor
-RUN yum -y update && yum -y install ca-certificates openssh openssh-server openssh-clients openssl-libs curl git
+FROM alpine as sensor
+COPY --from=base /usr/share/zoneinfo /usr/share/zoneinfo
+COPY --from=base /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
If I remember correctly, we need the git and other utilities for sensor to pull workflow definitions from git.

Reply:
ahhh, I totally missed that..... is `curl` required?

Reply:
Update. Verified pulling from git works with both `git://` and `https://`.